hp-markup.tex

```diff
@@ -28,58 +28,79 @@
 % Newspaper headlines
 \newcommand{\inlineheadline}[1]{\textsc{#1}}
 \newenvironment{headlines}{
-  \newcommand{\header}[1]{\begin{SingleSpace}\upshape ##1\end{SingleSpace}}
-  \let\hmorSavedLabel\label
-  \renewcommand{\label}[1]{{\upshape\itshape ##1}}%
-  \begin{Spacing}{0.75}
-  \begin{center}
-  \scshape
-  \nonzeroparskip
+\newcommand{\header}[1]{\begin{SingleSpace}\upshape ##1\end{SingleSpace}}
```

Comment:
Please can you restore the indentation in this section of code?

Reply:
ok, reverted the indent in this file